src/include/daos_task.h

```diff
@@ -241,6 +242,12 @@ typedef struct {
 	d_rank_list_t		*failed;
 } daos_pool_replicas_t;
 
+typedef struct {
```

Comment:
(style)  do not add new typedefs

Reply:
I would like to disregard this style warning in this patch because the newly introduced typedef / struct is in the same style as the other definitions in the file.